docs/navigation/anchor.mdx

```diff
@@ -1 +1,82 @@
-anchor page
\ No newline at end of file
+---
+product: dd360-ds
+title: React Anchor component
+components: Anchor
+---
+
+<br />
+<br />
+
+# <span name="floating-nav">Anchor</span>
+
+With the Anchor component you can create links with customizable properties and support for custom components.
+
+<br />
+
+## <span name="floating-nav">Imports</span>
```

Comment:
You can import by path too, like this:

```js
import Anchor from 'dd360-ds/Anchor'
```
And i think should recommend to users to use this alternative because they can reduce the bundle of the application